api/src/main/java/org/pmiops/workbench/google/CloudBillingClientImpl.java

```diff
@@ -0,0 +1,32 @@
+package org.pmiops.workbench.google;
+
+import com.google.api.services.cloudbilling.Cloudbilling;
+import com.google.api.services.cloudbilling.model.BillingAccount;
+import java.io.IOException;
+import javax.inject.Provider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CloudBillingClientImpl implements CloudBillingClient {
+  private final Provider<Cloudbilling> endUserCloudBillingProvider;
+  private final GoogleRetryHandler retryHandler;
+
+  @Autowired
+  public CloudBillingClientImpl(
+      Provider<Cloudbilling> endUserCloudBillingProvider, GoogleRetryHandler retryHandler) {
```

Comment:
does this need a qualifier? its named as an "endUser" billing provider but I don't see a qualifier to make it so.

Reply:
ah, good catch!